### PR TITLE
Keep linked comments visible in `conversation-activity-filter`

### LINKED ISSUE #5390: Exclude explicitly linked comments in `conversation-activity-filter`
### Description

Like #5334, but for `conversation-activity-filter`. E.g. in https://github.com/refined-github/refined-github/issues/4008#issuecomment-996659891, I linked an [earlier comment](https://github.com/refined-github/refined-github/issues/4008#issuecomment-824729016) that is now hidden. While the filter is active, I have no idea what I linked :sweat_smile: 

With #5562, explicitly linked comments are excluded on page load, so clicking this link shows the comment: https://github.com/refined-github/refined-github/issues/4008#issuecomment-824729016
However, clicking an in-page link doesn't work.

### Reproduction

1. Go to https://github.com/refined-github/refined-github/issues/4008
2. Load hidden timeline items
  grafik
3. Append this hash `#issuecomment-824729016` to the URL

… or …

1. Go to https://github.com/refined-github/refined-github/issues/4008#issuecomment-996659891
2. Load hidden timeline items
  grafik
3. Scroll down to my comment and click this link there (it's currently hidden as well :slightly_frowning_face:):
  grafik

### PR DESCRIPTION
**Pull request by jopemachine**
Fixes #5390.

## Test URLs

1. Go to https://github.com/refined-github/refined-github/issues/4008#issuecomment-996659891

2. Click the hidden comment and click the link (https://github.com/refined-github/refined-github/issues/4008#issuecomment-824729016)

### CODE REVIEW

File: source/features/conversation-activity-filter.tsx [hunk 1]
```diff
@@ -90,6 +90,14 @@ function processPage(): void {
 	}
 }
 
+function revealCollapsedComment(): void {
+	for (const item of select.all(`.js-timeline-item:is(.${collapsedClassName})`)) {
+		if (location.hash.startsWith('#issuecomment-') && select.exists(location.hash, item)) {
+			item.classList.remove(collapsedClassName);
+		}
+	}
```
Comment (cheap-glitch) [inline source/features/conversation-activity-filter.tsx:98]: This should probably be the opposite:

1. Select the targeted item (if it exists)
2. Go back up the DOM (using `closest`) and remove the class on the parent `.js-timeline-item` element

**Review by cheap-glitch — APPROVED** [review]
https://user-images.githubusercontent.com/46634000/163716313-44b9c802-d6e9-4f3b-8414-77b7805f7ed3.mp4

File: source/features/conversation-activity-filter.tsx [hunk 2]
```diff
@@ -221,6 +227,7 @@ async function init(): Promise<void> {
 		applyState(state);
 	}
 
+	window.addEventListener('hashchange', uncollapseTargetedComment);
```
Comment (fregante) [inline source/features/conversation-activity-filter.tsx:230]: This PR can be merged as is but in the other open PR we can add an AbortSignal here

**Review by fregante — APPROVED** [review]

File: source/features/conversation-activity-filter.tsx (hunk 2 above)
Comment (jopemachine) [reply source/features/conversation-activity-filter.tsx:230]: > This PR can be merged as is but in the other open PR we can add an AbortSignal here

I think the PR is maybe this one (https://github.com/refined-github/refined-github/pull/5523/files/d1511cb12654a480622e643c85f1dec87fef9d23).

Should I add `signal` like the below code in this PR?

```ts
async function init(signal: AbortSignal): Promise<void> {
	const state = minorFixesIssuePages.some(url => location.href.startsWith(url))
		? 'hideEventsAndCollapsedComments' // Automatically hide resolved comments on "Minor codebase updates and fixes" issue pages
		: 'default';

	await addWidget('#partial-discussion-header .gh-header-meta :is(clipboard-copy, .flex-auto)', state);
	await addWidget('#partial-discussion-header .gh-header-sticky :is(clipboard-copy, relative-time)', state);

	if (state !== 'default') {
		applyState(state);
	}

	// To do: when `:has` is available, replace below logic with css (#5390)
	window.addEventListener('hashchange', revealCollapsedComment, {signal});

	document.body.addEventListener('keypress', runShortcut, {signal});
}
```

**Pull request merged**